src/m365/teams/commands/team/team-add.ts

```diff
@@ -187,6 +320,48 @@ class TeamsTeamAddCommand extends GraphCommand {
 
     return group!;
   }
+
+  private async retrieveMembersToAdd(members: TeamMember[], role: string, emails?: string, ids?: string, userNames?: string): Promise<any[]> {
```

Comment:
If we're working on the members array that we pass here through args, then we don't need to return it as this method's output, because we get it by reference. So any update that we do on it, is automatically available to the caller of this method and which passed the array to it. Let's revisit the design of this method.